samples/Commerce/Commerce.UI/Views/FilterPage.xaml

```diff
@@ -36,70 +36,70 @@
 				<utu:AutoLayout Spacing="16"
 								Padding="16,16,16,16"
 								utu:AutoLayout.CounterAlignment="Stretch">
-					<TextBlock Foreground="{StaticResource MaterialOnBackgroundBrush}"
+					<TextBlock Foreground="{StaticResource OnBackgroundBrush}"
 							   x:Uid="Body_Categories"
 							   Text="Categories"
-							   Style="{StaticResource MaterialSubtitle2}" />
+							   Style="{StaticResource TitleMedium}" />
 					<utu:AutoLayout Spacing="8"
 									Orientation="Horizontal">
 						<utu:Chip Content="Shoes"
 								  x:Uid="Body_Shoes"
 								  IsChecked="{Binding Filter.Shoes, Mode=TwoWay}"
-								  Style="{StaticResource MaterialOutlinedFilterChipStyle}" />
+								  Style="{StaticResource OutlinedFilterChipStyle}" />
 						<utu:Chip Content="Accessories"
 								  x:Uid="Body_Accessproes"
 								  IsChecked="{Binding Filter.Accessories, Mode=TwoWay}"
-								  Style="{StaticResource MaterialOutlinedFilterChipStyle}" />
+								  Style="{StaticResource OutlinedFilterChipStyle}" />
 						<utu:Chip Content="Headwear"
 								  x:Uid="Body_Headwear"
 								  IsChecked="{Binding Filter.Headwear, Mode=TwoWay}"
-								  Style="{StaticResource MaterialOutlinedFilterChipStyle}" />
+								  Style="{StaticResource OutlinedFilterChipStyle}" />
```

Comment:
No more Outlined, it's either FilterChipStyle or ElevatedFilterChipStyle

https://m3.material.io/components/chips/guidelines#8d453d50-8d8e-43aa-9ae3-87ed134d2e64